alfalfa_worker/add_site/add_site.py

```diff
@@ -99,14 +99,15 @@ def os_files_final_touches_and_upload(self):
         Make unique ids and replace site_id.  Upload to mongo and filestore.
         :return:
         """
-        # TODO: Why exactly are the following two needed?
+        # This is neccessary to avoid avoid duplicates in the case that the same osm / osw is uploaded multiple times
+        # In alfalfa multiple uploades are treated as unique "sites"
         make_ids_unique(self.upload_id, self.report_haystack_json, self.report_mapping_json)
         replace_site_id(self.upload_id, self.report_haystack_json, self.report_mapping_json)
 
         # Upload to mongo & filestore, clean local directory
         self.add_to_mongo_and_filestore()
 
-    def add_to_mongo_and_filestore(self):
+    def add_to_mongo(self, site_ref, points_json):
```

Comment:
@kbenne `add_to_mongo_and_filestore` is still called in `add_fmu` function. should we just update the function called there?